### PR TITLE
Update test.c

### PR DESCRIPTION
**Pull request by dlemrry2**
added something. please review it

### CODE REVIEW

File: test.c [hunk 1]
```diff
@@ -5,3 +5,5 @@
 55555
 6666
 7777
+
+edit, please review
```
Comment (dlemrry) [inline test.c:9]: add more number

**Review by dlemrry — CHANGES_REQUESTED** [review]
review checked

File: test.c (hunk 1 above)
Comment (dlemrry2) [reply test.c:11]: review again please

**Review by dlemrry — APPROVED** [review]
review finished

**Pull request merged**